File: Bars.Gkh/Entities/Dicts/ContractService/AgreementContractService.cs
namespace Bars.Gkh.Entities.Dicts.ContractService
{
    using Bars.Gkh.Enums;

    /// <summary>
    /// Услуги/Работы по ДУ (договорам управления)
    /// </summary>
    public class AgreementContractService : ManagementContractService
    {
        /// <summary>
        /// Назначение работ
        /// </summary>
        public virtual WorkAssignment WorkAssignment { get; set; }

        /// <summary>
        /// Тип работ
        /// </summary>
        public virtual TypeWork TypeWork { get; set; }
    }
}

File: Bars.Gkh/libs/B4/model/Dict/contractservice/AgreementContractService.js
Ext.define('B4.model.dict.contractservice.AgreementContractService', {
    extend: 'B4.model.dict.contractservice.ManagementContractService',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'agreementContractService'
    },
    fields: [
        { name: 'WorkAssignment' },
        { name: 'TypeWork' }
    ]
});